pageserver/src/tenant/delete.rs

```diff
@@ -136,7 +136,9 @@ async fn schedule_ordered_timeline_deletions(
     let mut already_running_deletions = vec![];
 
     for (timeline_id, _) in sorted.into_iter().rev() {
-        if let Err(e) = DeleteTimelineFlow::run(tenant, timeline_id, true).await {
+        let span = tracing::info_span!("timeline_delete", %timeline_id);
+        let fut = DeleteTimelineFlow::run(tenant, timeline_id, true).instrument(span);
```

Comment:
I think it would be better to return a result here instead of a future and then awaiting it later.

Reply:
I was just trying to keep under the max line length, which the `.await` might still be... I don't see a big difference, can add it for a new test run.

Reply:
It does not fit on single line.

Do you approve a version which would have `let res = fut.await;`? :)